core/src/main/scala/org/apache/spark/scheduler/ResultTask.scala

```diff
@@ -51,8 +51,12 @@ private[spark] class ResultTask[T, U](
     locs: Seq[TaskLocation],
     val outputId: Int,
     localProperties: Properties,
-    metrics: TaskMetrics)
-  extends Task[U](stageId, stageAttemptId, partition.index, metrics, localProperties)
+    metrics: TaskMetrics,
+    jobId: Option[Int] = None,
```

Comment:
you need to update the description of this class to have new params and descriptions